pyccel/ast/core.py

```diff
@@ -3374,12 +3362,10 @@ def __reduce_ex__(self, i):
 
     # TODO
     def check_pure(self):
-        sys.exit("check_pure in functiondef\n")
         raise NotImplementedError('')
 
     # TODO
     def check_elemental(self):
-        sys.exit("check_elemental in functiondef\n")
         raise NotImplementedError('')
```

Comment:
@saidctb Presumably these functions can also be removed as they are not even implemented?

Reply:
I agree. These two functions must be removed!

Reply:
yes